mothernet/priors/step_function_prior.py

```diff
@@ -0,0 +1,46 @@
+import torch
+
+from mothernet.distributions import parse_distributions, sample_distributions
+from mothernet.utils import default_device
+
+
+class StepFunctionPrior:
+    def __init__(self, config=None):
+        self.config = parse_distributions(config or {})
+
+    def _get_batch(self, batch_size, n_samples, num_features, device=default_device, num_outputs=1, epoch=None,
+                  single_eval_pos=None):
+        # small wrapper to sample from the prior but also return the individual step functions.
+        with torch.no_grad():
+            hypers = sample_distributions(self.config)
+            if hypers['sampling'] == 'uniform':
+                x = torch.rand(batch_size, n_samples, num_features, device=device)
+            else:
+                x = torch.randn(batch_size, n_samples, num_features, device=device)
+
+            # Per element in batch create a random step function per feature and add them across samples.
+            start = torch.randint(0, 2, size=(batch_size, num_features), device=device).unsqueeze(dim=1)
```

Comment:
can you maybe name this "up_or_down" or something? It took me a second. Or maybe ``start_low``